### PR TITLE
Adding documentation for Cassandra deployment on k8s with PX

### PR DESCRIPTION
**Pull request by Hrishike**
Documentation which talks about deployment of a Cassandra statefulset with PX on a k8s cluster. 
Sections covered. 
- Dynamic Volume provisioning using PVC with PX as the provisioner
- Scaling of the statefulset and a k8s node and how PX reacts to it. 
- Failovers
    a) Pod Failover
    b) Node Failover

### CODE REVIEW

File: scheduler/kubernetes/cassandra-k8s.md [hunk 1]
```diff
@@ -0,0 +1,704 @@
...
+        Bytes used               :  33 MiB
+        Replica sets on nodes:
+                Set  0
+                        Node     :  10.142.0.4
+                        Node     :  10.142.0.3
+
+```
+
+### Scaling
+Portworx runs as a Daemon set within Kubernetes. Hence when you add a node or a worker to your kuberentes cluster you do not need to run Portworx on it.
```
Comment (Hrishike) [inline scheduler/kubernetes/cassandra-k8s.md]: done.

File: scheduler/kubernetes/cassandra-k8s.md [hunk 2]
```diff
@@ -0,0 +1,704 @@
...
+  "PodIP": "10.0.64.2"
+}  
+{
+  "name": "cassandra-2",
+  "hostname": "k8s-1",
+  "hostIP": "10.142.0.4",
+  "PodIP": "10.0.192.3"
+}
+```
+Portworx applies labels to the node objects where it places the volumes. so in this case ```k8s-0``` node which runs ```cassandra-1``` pod has the label applied to it as ```cassandra-data-cassandra-1=true```. Check output of the command below.
```
Comment (Hrishike) [inline scheduler/kubernetes/cassandra-k8s.md]: Removed them now.

File: _data/sidebars/home_sidebar.yml [hunk 3]
```diff
@@ -16,6 +16,10 @@
         url: /scheduler/kubernetes/install.html
       - title: Upgrade
         url: /scheduler/kubernetes/upgrade.html
+      - title: Stateful frameworks
```
Comment (Hrishike) [inline _data/sidebars/home_sidebar.yml]: Sure. Just fyi though, the DC/OS submenu also might need to be changed.

Comment (venkatpx) [inline _data/sidebars/home_sidebar.yml]: DCOS doesn't need to change.

**Pull request merged**